wp-content/themes/access/single-programs.php

```diff
@@ -37,15 +37,18 @@
 /**
  * Get alerts
  */
-
-$alerts = Timber::get_posts(array(
-  'post_type' => 'alert',
-  'posts_per_page' => -1
-));
-
-$context['alerts'] = array_filter($alerts, function($p) {
-  $flags = ['programs', 'single'];
-  return count(array_intersect(array_values($p->custom['location']), $flags)) === count($flags);
-});
+if (get_field('alert')) {
+  $context['alerts'] = get_field('alert');
+} else {
+  $alerts = Timber::get_posts(array(
+    'post_type' => 'alert',
+    'posts_per_page' => -1
+  ));
+
+  $context['alerts'] = array_filter($alerts, function($p) {
+    $flags = ['programs', 'single'];
+    return count(array_intersect(array_values($p->custom['location']), $flags)) === count($flags);
+  });
+}
 
 Timber::render($templates, $context);
```

Comment:
I think you need to update your branch... These changes should already be there.